yarn-project/end-to-end/src/e2e_fees.test.ts

```diff
@@ -275,6 +278,114 @@ describe('e2e_fees', () => {
     );
   });
 
+  it('fails transaction that error in setup', async () => {
+    const OutrageousPublicAmountAliceDoesNotHave = 10000n;
+    // const PublicMintedAlicePublicBananas = 1000n;
+    const FeeAmount = 1n;
+    const RefundAmount = 2n;
+    const MaxFee = FeeAmount + RefundAmount;
+    const { wallets } = e2eContext;
+
+    // simulation throws an error when setup fails
+    await expect(
+      bananaCoin.methods
+        .transfer_public(aliceAddress, sequencerAddress, OutrageousPublicAmountAliceDoesNotHave, 0)
+        .send({
```

Comment:
Could call `simulate` directly to make it match the comment above.

```suggestion
        .simulate({
```

Reply:
My thinking was that it was simulating the public part since `skipPublicSimulation` is not set.